app/build.gradle

```diff
@@ -16,6 +16,10 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
+    compileOptions {
+        sourceCompatibility = 1.8
```

Comment:
what's this mean?

Reply:
Needed to upgrade the Java version to be able to use lambda functionality which is supported from that version and higher.